modules/checklists/assets/js/meta-box.js

```diff
@@ -777,6 +823,111 @@
         update();
     }
 
+    /*----------  Internal Links ----------*/
+    var lastCount = 0;
+    if (PP_Checklists.is_gutenberg_active()) {
+        /**
+         * For Gutenberg
+         */
+        if ($('#pp-checklists-req-internal_links').length > 0) {
+            wp.data.subscribe(
+                function () {
+                    // @todo: why does Multiple Authors "Remove author from new posts" setting cause this to return null?
```

Comment:
Please, remove this comment it is out of context here

Reply:
Hi, thank you. I've updated it now.

Reply:
Cool, thanks 👍